### PR TITLE
test(sbb-datepicker): attempt to stabilize visual regression tests

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

**Review by DavideMininni-Fincons — APPROVED** [review]
Looks good;
should the standalone cases in prev-next buttons be removed, as in the toggle?

File: src/elements/datepicker/datepicker-next-day/datepicker-next-day.visual.spec.ts [hunk 1]
```diff
@@ -1,64 +1,62 @@
...
 import '../datepicker.js';
 import '../../form-field.js';
 
 describe(`sbb-datepicker-next-day`, () => {
-  const cases = [
-    { name: 'no value', value: null },
-    { name: 'with value', value: '15.02.2023' },
-  ];
+  const cases = {
+    negative: [true, false],
```
Comment (DavideMininni-Fincons) [inline src/elements/datepicker/datepicker-next-day/datepicker-next-day.visual.spec.ts:18]: i suppose negative=true would require a black background color and a white outline color

File: src/elements/datepicker/datepicker-next-day/datepicker-next-day.visual.spec.ts [hunk 2]
```diff
@@ -1,64 +1,62 @@
+import { sendKeys } from '@web/test-runner-commands';
```
Comment (DavideMininni-Fincons) [inline src/elements/datepicker/datepicker-next-day/datepicker-next-day.visual.spec.ts:1]: negative=false-value=null_with_form-field_focus is different between prev and next button

Comment (jeripeierSBB) [reply src/elements/datepicker/datepicker-next-day/datepicker-next-day.visual.spec.ts:1]: I think I just resolved that. Let's see

**Comment by DavideMininni-Fincons** [comment]
LGTM

**Pull request merged**